src/mlpack/tests/q_learning_test.cpp

```diff
@@ -465,4 +465,43 @@ BOOST_AUTO_TEST_CASE(CartPoleWithNStepPrioritizedDQN)
   BOOST_REQUIRE(converged);
 }
 
+//! Test Categorical DQN in Cart Pole task.
+BOOST_AUTO_TEST_CASE(CartPoleWithCategoricalDQN)
+{
+  // It isn't guaranteed that the network will converge in the specified number
+  // of iterations.
+  bool converged = false;
+  for (size_t trial = 0; trial < 3; ++trial)
+  {
+    Log::Debug << "Trial number: " << trial << std::endl;
```

Comment:
I guess we can remove the output here.

Reply:
Hmm.. Its present in other tests as well, and sometimes I find it useful while running tests locally. But I'm fine with removing it.

Reply:
If it helps, let's keep it.